README.md

```diff
@@ -25,13 +25,39 @@ Or install it yourself as:
 ```ruby
 require 'zeus/api_client'
 zeus_client = Zeus::APIClient.new({
-    :access_token => "your_token_here",
+    :token => "your_token_here",
```

Comment:
what versions of ruby are we using?

in newer version the standard is token: "your token here"